src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -80,7 +80,7 @@ public void Process(TrackedDownload trackedDownload, bool ignoreWarnings = false
                     return;
                 }
 
-                var artist = trackedDownload.RemoteAlbum.Artist;
+                var artist = trackedDownload.RemoteAlbum?.Artist;
```

Comment:
FYI, this was changed to trackedDownload many moons ago as a quick hack because we didn't have a good getArtist method in the parsing service that would parse the download title accurately. We could look at changing this back to something like Sonarr that will parse the title here instead of relying on the remoteAlbum. `var series = _parsingService.GetSeries(trackedDownload.DownloadItem.Title);`

Reply:
Tracing it back quickly it seems to come down to almost the same thing (i.e. `TrackedDownloadService` has parsed the album title and called `GetArtist`, but the version called by `TrackedDownloadService` also takes into account `searchCriteria` if it exists.  So I'd be inclined to leave it as is unless I've missed something.